app/assets/javascripts/proxy_status/logs.js

```diff
@@ -52,6 +52,8 @@ function activateLogsDataTable() {
     if (link.data('message')) modal.find('#modal-bt-message').text(link.data('message'));
     if (link.data('backtrace')) modal.find('#modal-bt-backtrace').text(link.data('backtrace'));
   })
+  // Activate tooltips for fields with ellipsis
+  tfm.tools.activateTooltips();
```

Comment:
you can limit this to just the table by passing its id as a parameter, i.e. `tfm.tools.activateTooltips('#table-proxy-status-logs')`